### PR TITLE
[DOCS] Add details of team calls

### LINKED ISSUE #3850: Improve the contributors guide
Some good inspiration to read first:
- https://github.com/ethereum/remix-ide/blob/d5aec6736dafd5050119aafffe5ac1e353943457/team-best-practices.md
- https://github.com/pipermerriam/ethereum-dev-tactical-manual

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Description

Closes https://github.com/ethereum/solidity/issues/3850 where we have now undertaken most of the tasks here already, but I felt we could add details of our team calls, not necessary though. The testing tasks remain to be completed, but am waiting for changes before diving back into that.

### Checklist
- [x] Code compiles correctly
- [x] All tests are passing
- [x] New tests have been created which fail without the change (if possible)
- [x] README / documentation was extended, if necessary
- [x] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -21,6 +21,17 @@ In particular, we need help in the following areas:
...
+Team Calls
+==========
+
+If you have issues or pull requests to discuss, or are interested in hearing what
+the team and contributors are working on, we have two weekly team calls:
+
+- Monday at 12pm CET
+- Wednesday at 3pm CET
+
+Both calls take place on `this Google Hangout <https://hangouts.google.com/hangouts/_/ethereum.org/solidity-weekly>`_.
```
Comment (chriseth) [inline docs/contributing.rst]:
```suggestion
Both calls take place on `Google Hangouts <https://hangouts.google.com/hangouts/_/ethereum.org/solidity-weekly>`_.
```

File: docs/contributing.rst [hunk 2]
```diff
@@ -21,6 +21,17 @@ In particular, we need help in the following areas:
 
 Please note that this project is released with a `Contributor Code of Conduct <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_. By participating in this project - in the issues, pull requests, or Gitter channels - you agree to abide by its terms.
 
+Team Calls
+==========
+
+If you have issues or pull requests to discuss, or are interested in hearing what
+the team and contributors are working on, we have two weekly team calls:
```
Comment (chriseth) [inline docs/contributing.rst]:
```suggestion
the team and contributors are working on, you can join our public team calls:
```

**Comment by Marenz** [comment]
please squash the commits into one.

**Comment by bshastry** [comment]
Looks like the ossfuzz build CI is failing because the PR is not rebased, could you please rebase as well?

**Comment by ChrisChinchilla** [comment]
It had sat here for a bit… @Marenz and @bshastry squashed and rebased

**Review by Marenz — APPROVED** [review]

**Pull request merged**